### PR TITLE
Added how-to guide: k-fold cross-validation

### LINKED ISSUE #28: Cross validation guide
Ideas on how to do cross-validation: https://github.com/pytorch/ignite/issues/1384

### PR DESCRIPTION
**Pull request by LiviaCavalcanti**
Resolves #28. 
A how-to guide of training model with cross-validation using the k-fold technique to estimate its performance.  For data split into the folds, the scikit-learn library was used.

### CODE REVIEW

File: how-to-guides/07-cross-validation.ipynb [hunk 1]
```diff
@@ -0,0 +1,1931 @@
...
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "outputId": "f222917d-176c-4a3c-a56d-bf84bb872a21"
+      },
+      "source": [
+        "acc_sum = 0\n",
+        "for n_fold in range(len(results_per_fold)):\n",
+        "  current_fold = results_per_fold[n_fold]\n",
+        "  print(f\"Validation Results - Epoch[{n_fold + 1}] Avg accuracy: {current_fold[1][2]['Accuracy']:.2f} Avg loss: {current_fold[1][2]['Loss']:.2f}\")\n",
```
Comment (sdesrozis) [inline how-to-guides/07-cross-validation.ipynb:1910]:
```suggestion
        "  print(f\"Validation Results - Fold[{n_fold + 1}] Avg accuracy: {current_fold[1][2]['Accuracy']:.2f} Avg loss: {current_fold[1][2]['Loss']:.2f}\")\n",
```

Comment (sdesrozis) [inline how-to-guides/07-cross-validation.ipynb:1910]: `current_fold[1][2]["Accuracy"]` means getting the Accuracy score of the validation ([1]) at epoch 3 ([2]). Maybe it worth mention that.

**Comment by sdesrozis** [comment]
@LiviaCavalcanti Thank you for this great contribution ! I left a few comments.

**Comment by vfdev-5** [comment]
@LiviaCavalcanti thanks a lot for the PR ! Looks good to me
